source/community/reactnative/src/components/containers/container-wrapper.js

```diff
@@ -47,15 +60,125 @@ export class ContainerWrapper extends React.PureComponent {
     getBackgroundImageContainer = () => {
         if (Utils.isString(this.payload.backgroundImage)) {
             this.payload.backgroundImage = {
-                url: this.payload.backgroundImage
-            }
+                url: this.payload.backgroundImage,
+            };
         }
+        // padding
+		const padding = this.props.configManager.hostConfig.getEffectiveSpacing(Enums.Spacing.Padding);
         return (
             <React.Fragment>
-                <BackgroundImage backgroundImage={this.payload.backgroundImage} />
-                {this.props.children}
-            </React.Fragment >
+                <BackgroundImage
+                    backgroundImage={this.payload.backgroundImage}
+                />
+                <View style={{paddingTop: padding, paddingBottom: padding}}>
+                    {this.props.children}
+                </View>
+            </React.Fragment>
+        );
+    };
+
+    /**
+     * @description The method will return true, if any of the parent containers having the style applied other than default.
+     */
+    hasParentStyle(payload) {
+        if (payload.parent) {
+            if (
+                payload.parent['style'] &&
+                payload.parent['style'] != Enums.ContainerStyle.Default
+            ) {
+                return true;
+            }
+        } else {
+            this.hasParentStyle(payload.parent);
+        }
+    }
+
+    /**
+     * @description The method will return true, if any of the parent containers is column.
+     */
+    applyBleedMarginHorizontal(computedStyles) {
+        const padding = this.props.configManager.hostConfig.getEffectiveSpacing(
+            Enums.Spacing.Padding,
         );
+        if (this.payload.type == Constants.TypeColumn) {
+            //If parent that means columnset have style other than default, then we need to apply padding as -padding otherwise padding as 0
+            const marginValue =
+                this.payload.parent['style'] &&
+                this.payload.parent['style'] != Enums.ContainerStyle.Default
+                    ? 0
+                    : -padding;
+            if (this.props.isFirst) {
```

Comment:
Can isFirst and isLast both be true? in case of only one view. In such case, we need to add margin on both sides. So, Is it better to replace else if with just if?

Reply:
Resolved